src/daffodilDebugger/utils.ts

```diff
@@ -78,9 +90,11 @@ export async function runDebugger(
     scriptPath,
     artifact.scriptName,
     createTerminal,
-    shellArgs(artifact.scriptName, serverPort),
+    shellArgs(serverPort),
     {
       DAFFODIL_DEBUG_CLASSPATH: daffodilDebugClasspath,
+      LOG_LEVEL: debuggerLogLevel,
+      LOG_FILE: debuggerLogFilePath,
```

Comment:
```suggestion
      DAFFODIL_DEBUGGER_LOG_LEVEL: debuggerLogLevel,
      DAFFODIL_DEBUGGER_LOG_FILE: debuggerLogFilePath,
```
Env vars should be namespaced to avoid clobbering.